std/experimental/allocator/package.d

```diff
@@ -2263,6 +2263,17 @@ if (isInputRange!R)
     assert(arr == [1, 2, 3, 1, 2, 3, 4]);
 }
 
+private bool implExpandArray(Allocator)(auto ref Allocator alloc, ref void[] array,
+        size_t delta)
+{
+    static if (__traits(hasMember, Allocator, "expand"))
+    {
+        return alloc.expand(buf, T.sizeof * delta);
+    } else {
```

Comment:
Braces and `else` should be on their own lines (https://dlang.org/dstyle.html#phobos_brackets)